tests/snapshots/tests.contrib.django.test_django_snapshots.test_psycopg_query_default.snap

```diff
@@ -0,0 +1,38 @@
+[[{"name" "postgres.query"
```

Comment:
are we going to have issues with using the same operation name?

is this changing at all from what we do today?

Reply:
this operation name is set within the django integration here:

https://github.com/DataDog/dd-trace-py/blob/c09c4dddab5cbc4222233a9ab0f286eaeac5261a/ddtrace/contrib/django/patch.py#L79https://github.com/DataDog/dd-trace-py/blob/c09c4dddab5cbc4222233a9ab0f286eaeac5261a/ddtrace/contrib/django/patch.py#L84
https://github.com/DataDog/dd-trace-py/blob/c09c4dddab5cbc4222233a9ab0f286eaeac5261a/ddtrace/contrib/dbapi/__init__.py#L40-L41